app/src/main/java/to/dev/dev_android/util/AndroidWebViewBridge.kt

```diff
@@ -17,4 +19,21 @@ class AndroidWebViewBridge {
     fun logError(errorTag: String, errorMessage: String) {
         Log.e(errorTag, errorMessage)
     }
+
+    /**
+     * This method is used to open the native share-sheet of Android when simple text is to be
+     * shared from the web-view.
+     */
+    @JavascriptInterface
+    fun shareText(text: String) {
+        val shareIntent = Intent.createChooser(
+            Intent().apply {
+                action = Intent.ACTION_SEND
+                putExtra(Intent.EXTRA_TEXT, text)
+                type = "text/plain"
+            },
+            null
+        )
+        context.startActivity(shareIntent)
+    }
```

Comment:
`AndroidWebViewBridge.kt` has seen some changes in the past few months, everything is hooked up now from `MainActivity.kt`.

This function is all we would need in order to make this work (and maybe the imports 😄)

Reply:
Thanks for the directions 🙂